File: swift-ios-developers/2-swift-ios-developers-m2-exercise-files/demo1_after/ObjCSwiftDemo/ObjCSwiftDemo/ViewController.m
//
//  ViewController.m
//  ObjCSwiftDemo
//
//  Created by Jon Flanders on 1/30/16.
//  Copyright © 2016 Flounderware. All rights reserved.
//

#import "ViewController.h"
#import "ObjCSwiftDemo-Swift.h"

@interface ViewController ()

@end

@implementation ViewController
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender{
	UIViewController* vc = segue.destinationViewController;
	SwiftViewController* svc = vc;
	svc.data = @"Hi From Objective-C";
	
	//svc.label.text = @"Hi From ObjC";
}
-(IBAction)back:(UIStoryboardSegue*)sender{
	
}
- (void)viewDidLoad {
	[super viewDidLoad];
	// Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
	[super didReceiveMemoryWarning];
	// Dispose of any resources that can be recreated.
}

@end

File: swift-ios-developers/3-swift-ios-developers-m3-exercise-files/step3_storyboard/SwiftExample/SwiftExample/ViewController.swift
//
//  ViewController.swift
//  SwiftExample
//
//  Created by Jon Flanders on 2/6/16.
//  Copyright © 2016 Flounderware. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

	override func viewDidLoad() {
		super.viewDidLoad()
		// Do any additional setup after loading the view, typically from a nib.
		//some sort of Dependency Injection
		let computerController = MockComputerController()
		var computerAddresses = [String]()
		for _ in 0...25{
			computerAddresses.append("localhost")
		}
		computerController.getComputer(computerAddresses) { (computer, err) -> Void in
			
		}
		
	}

	override func didReceiveMemoryWarning() {
		super.didReceiveMemoryWarning()
		// Dispose of any resources that can be recreated.
	}


}

